src/SprykerMiddleware/Zed/Process/Business/Mapper/Mapper.php

```diff
@@ -178,7 +178,7 @@ protected function filterArray(array $array, array $value): array
      */
     protected function prepareResult(array $payload): array
     {
-        if ($this->map->getStrategy() === ProcessConstants::MAPPER_STRATEGY_COPY_UNKNOWN) {
+        if ($this->mapperConfigTransfer->getStrategy() === ProcessConstants::MAPPER_STRATEGY_COPY_UNKNOWN) {
```

Comment:
Please use ProcessConstants only for env-specific configuration.
For the rest please use ProcessConfig (Zed or Shared).